build/dsc/manager.rb

```diff
@@ -188,6 +188,8 @@ def document_types(markdown_file_path, dsc_types)
           end
           file.write("\n")
         end
+        
+        file.write("\nGit reference of DSC used to create types: #{ENV['DSC_REF']}")        
```

Comment:
The import directory is purged of git history as part of https://github.com/puppetlabs/puppetlabs-dsc/blob/master/build/dsc.rake#L70 .  I can modify it to only add that footnote if `ENV['DSC_REF']` is defined.